### ISSUE TITLE
[Feature Request] Support Date Time Range Picker

### ISSUE THREAD

**Issue by frogdevvn** [dan-lee/timescape#2]
Thanks for great repo. I would like to ask if you can add support for date time range picker like React-Aria: https://react-spectrum.adobe.com/react-aria/DateRangePicker.html

**Comment by dan-lee** [comment]
Hey @frogdevvn, thanks that's a great idea 💡! I'll think a bit about how this could be best implemented with the current library

**Comment by gmwill934** [comment]
I second this! :)

**Comment by dan-lee** [comment]
What do you think about this API?

```tsx
const { from, to, isValid } = useTimescapeRange({
  minDate: new Date('2020-01-01'),
  maxDate: new Date('2024-01-01'),
  // ...
})

// ...

<div>
  <input {...from.getInputProps('days') />
  <input {...to.getInputProps('days') />
  {isValid ? '⛔️' : '✅'}
</div>
```

**Comment by dan-lee** [comment]
This has been released in [`v0.4.0`](https://github.com/dan-lee/timescape/releases/tag/timescape%400.4.0) 🚀
 
You can check it out here: https://timescape.daniellehr.de/?path=/story/root-timescape--range-calendar
